server/controllers/healthNews.js

```diff
@@ -0,0 +1,13 @@
+const { getNews } = require('../database/queries');
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
don't disable  eslint role

Reply:
i will not be using req in this route that's why i disabled this line only